docs/src/pages/[platform]/index.page.tsx

```diff
@@ -4,10 +4,10 @@ import { useRouter } from 'next/router';
 
 import { Footer } from '@/components/Layout/Footer';
 import { HeroSection } from '@/components/home/sections';
-import { isReactNativeEnabled } from '@/utils/featureFlags';
+import { IS_REACT_NATIVE_ENABLED } from '@/utils/featureFlags';
 
 import ReactHomePage from './index.react';
-import ReactNativeHomePage from './index.reactnative';
+import ReactNativeHomePage from './index.react-native';
```

Comment:
the `index.reactnative` file currently has `query: { platform = 'react' },`. Should be `query: { platform = 'react-native' },`.

Reply:
Oh interesting. That doesn't sound right though, how does it still work?

Reply:
Guessing we never end up needing the default param. I do think this is something we should address across the codebase fwiw